schema_based_on_diagram.sql

```diff
@@ -0,0 +1,39 @@
+-- create patients
+CREATE TABLE patients (
+id INT PRIMARY KEY,
+name VARCHAR(255),
+date_of_birth DATE );
+-- create medical histories
+CREATE TABLE medical_histories (
+id INT PRIMARY KEY,
+admitted_at TIMESTAMP,
+patient_id INT,
+status VARCHAR(60),
+FOREIGN KEY (patient_id) REFERENCES patients(id)
+);
+-- create invoices
+CREATE TABLE invoices (
+id INT PRIMARY KEY,
+total_amount DECIAML(5,2),
+generated_at TIMESTAMP,
+payed_at TIMESTAMP,
+medical_history_id INT
+FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
+ );
+-- create treatments
+CREATE TABLE treatments (
+id INT PRIMARY KEY,
+type VARCHAR(50),
+name VARCHAR(50)
+);
+-- create invoice items
```

Comment:
- [ ] It would be nice if you create indexes for all the foreign keys from the tables you have created to follow the project requirements 😊